packages/template-retail-react-app/app/components/item-variant/item-attributes.test.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import ItemVariantProvider from '@salesforce/retail-react-app/app/components/item-variant'
+import {renderWithProviders} from '@salesforce/retail-react-app/app/utils/test-utils'
+import {screen, waitFor} from '@testing-library/react'
+import ItemAttributes from '@salesforce/retail-react-app/app/components/item-variant/item-attributes'
+import {getDisplayVariationValues} from '@salesforce/retail-react-app/app/utils/product-utils'
+import PropTypes from 'prop-types'
+import {rest} from 'msw'
+import {
+    mockBundledProductItemsVariant,
+    mockProductBundleWithVariants,
+    mockProductBundle
+} from '@salesforce/retail-react-app/app/mocks/product-bundle'
+
+const MockedComponent = ({variant}) => {
+    return (
+        <ItemVariantProvider variant={variant}>
+            <ItemAttributes />
+        </ItemVariantProvider>
+    )
+}
+MockedComponent.propTypes = {
+    variant: PropTypes.object
+}
+
+beforeEach(() => {
+    global.server.use(
+        rest.get('*/products', (req, res, ctx) => {
+            return res(ctx.delay(0), ctx.status(200), ctx.json(mockProductBundleWithVariants))
+        }),
+        rest.get('*/products/:productId', (req, res, ctx) => {
+            return res(ctx.delay(0), ctx.status(200), ctx.json(mockProductBundle))
+        })
+    )
+})
+
+test('component renders product bundles with variant data', async () => {
+    renderWithProviders(<MockedComponent variant={mockBundledProductItemsVariant} />)
+    await waitFor(() => {
+        expect(screen.getByText(/selected options:/i)).toBeInTheDocument()
+    })
```

Comment:
```suggestion
    expect(await screen.findByText(/selected options:/i)).toBeInTheDocument()
```
[nit] I recently learned that `find*` is just a convenience method for `waitFor(() => get*())`!

Reply:
Tried using `expect(await screen.findByText(/selected options:/i)).toBeInTheDocument()`  and I get back an empty object instead of an `HTMLElement` 🤔🤔🤔 I am confusion as this was not what I was expecting

Reply:
@wjhsf I don't think that's valid. You could do something like 
```suggestion
    const something = screen.findByText(/selected options:/i) // already uses `await` implicitly
    await waitFor(() => {
        expect(something).toBeInTheDocument()
    })
```